index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Начальная страница -- Главная</title>
+    
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css">
+</head>
+<body>
+    <header class="header">
+
+        <img src="images/logo.png" class="logo" alt="logo">
+        <h1 class="heading">Главная страница</h1>
+
+        <button id="menu-button">
+
+            <div class="header-menu"> <img src="images/icon.png" alt="icon"></div>
+        </button>
+
+        <nav id="navbar" class="navbar">
+            <ul class="ul-menu">
+                <li class="li-menu-item">
+                    <a class="a-menu" href="index.html">Главная</a>
+                </li>
+                <li class="li-menu-item">
+                    <a class="a-menu" href="teams.html">Игра</a>
+                </li>
+                <li class="li-menu-item">
+                    <a class="a-menu" href="rules.html">Правила</a>
+                </li>
+            </ul>
+        </nav>
+
+        <div id="mobile-menu" class="mobile-menu">
+            <ul class="ul-menu">
+                <li class="li-menu-item">
+                    <a class="a-menu" href="index.html">Главная</a>
+                </li>
+                <li class="li-menu-item">
+                    <a class="a-menu" href="teams.html">Игра</a>
+                </li>
+                <li class="li-menu-item">
+                    <a class="a-menu" href="rules.html">Правила</a>
+                </li>
+            </ul>
```

Comment:
You should differentiate design via CSS, not by code duplication.

Reply:
Redone menu